helloworld/BUILD

```diff
@@ -23,6 +23,9 @@ resources(
 pex_binary(
     name="pex_binary",
     entry_point="main.py",
+
+    # needed when used in the Docker image, in case pants is running on a incompatible platform
+    platforms=['linux-x86_64-cp-38-cp38']
```

Comment:
Would it make sense to be able to "inject" or otherwise ensure that there is a specific value present for the `platforms` field of a `pex_binary` target.

I'm thinking that when packaging for the Docker build context, we could "know" what platform we want it packaged for..

Related pantsbuild/pants#13185.

Reply:
This means that you can't run the binary anymore if you're on macOS, which is a show stopper.

Reply:
I guess it is possible to add more platforms there, so Mac would still be supported?

But my q is after if we could leave this change out entirely, and add it as needed when packaging for the docker image..

All hypothetical as we'll close this particular PR. ;)

Reply:
With adding automatically, how would we know which Python version to target? I don't think static analysis of the Dockerfile would work

Reply:
> I don't think static analysis of the Dockerfile would work

No, that's problematic. I'd like a platform of just `"linux-x86_64"`, possibly with the `-cp` but not more than that.. hmm.

For specific cases, such when you use `FROM python:3.9` for instance, we can, but that's very specific.